code/modules/projectiles/ammo_datums.dm

```diff
@@ -1182,7 +1182,7 @@ datum/ammo/bullet/revolver/tp44
 	accuracy = 40
 	accurate_range = 20
 	max_range = 30
-	damage = 100
+	damage = 140
```

Comment:
Those are on impact hit the rockets also cause explosion on hit.

Reply:
This is only meant for impact.